server/api/tumdev/campus_backend.proto

```diff
@@ -532,13 +538,14 @@ enum Language {
   English = 1;
 }
 
-message ListStudentClubRequest {
+message ListStudentGroupRequest {
   // Language of the student clubs and categories
   // Defaults to german
   optional Language language = 1;
 }
-message ListStudentClubReply {
-  repeated StudentClubCollection collections = 1;
+message ListStudentGroupReply {
+  repeated StudentClubCollection student_clubs = 1;
+  repeated StudentCouncilCollection student_councils = 2;
```

Comment:
So what, we want to represent each of the councils separately? Why? It seems like a lot of additional complexity.

Reply:
I don't know what the best API is for this.
My rationale was that a student has >=1 student council (f.ex. ASTA, MPIC), some also have 
>> 1 if they are studying more than one Program.

Since I don't quite know how this matching should work, I have pushed the matching part off to another PR.